fxmanifest.lua

```diff
@@ -8,19 +8,27 @@ If you just want to change the postal file, **ONLY** change the above variable
 --]]
 fx_version 'cerulean'
 games { 'gta5' }
+lua54 "yes"
 
 author 'blockba5her'
-description 'This script displays a nearest postal next to where PLD would go and also has a command to draw a route to a specific postal'
-version '1.4.2'
+description 'This script displays the nearest postal next to map, and allows you to navigate to specific postal codes'
+version '1.5.0'
 url 'https://github.com/blockba5her/nearest-postal'
 
 client_scripts {
-	'config.lua',
-	'cl.lua'
+    'config.lua',
+
+    'cl.lua',
+    'cl_commands.lua',
+    'cl_render.lua',
+
+    -- uncomment to enable dev tools
+    --'cl_dev.lua',
```

Comment:
This is fine, but please update the `README.md` with the new instructions for enabling dev mode.